src/clj/krueger/routes/services.clj

```diff
@@ -48,17 +48,16 @@
     (auth/login email pass req))
 
   (POST "/api/register" req
-    :return auth/LoginResponse
-    :body-params [screnname :- s/Str
+    :return common/Success
+    :body-params [screenname :- s/Str
                   email :- s/Str
                   pass :- s/Str]
     :summary "user registration handler"
-    (auth/register!
-      {:screnname screnname
-       :email     email
-       :pass      pass}
-      req)
-    (ok {:result :ok}))
+    ; REVIEW: I'm not sure this gives the correct result if `register!` 500s
+    (ok {:result (auth/register! {:screenname screenname
```

Comment:
The default middleware [here](https://github.com/yogthos/krueger/blob/a8e3814c9be7ba7d3495a6b069ccf46ab8e54f06/src/clj/krueger/middleware.clj#L39) should handle that case gracefully.